sunpy/timeseries/tests/test_timeseries_factory.py

```diff
@@ -368,22 +368,30 @@ def test_invalid_manual_data():
     with pytest.raises(NoMatchError):
         sunpy.timeseries.TimeSeries(data, meta)
 
-
+@pytest.mark.filterwarnings('ignore:"silence_errors" was deprecated in version 5')
 def test_invalid_filepath():
     invalid_filepath = os.path.join(rootdir, 'invalid_filepath_here')
     with pytest.raises(ValueError, match='Did not find any files'):
         sunpy.timeseries.TimeSeries(invalid_filepath)
     # Now with silence_errors kwarg set
     with pytest.raises(ValueError, match='Did not find any files'):
         sunpy.timeseries.TimeSeries(invalid_filepath, silence_errors=True)
+    # Now with allow_errors kwarg set
+    with pytest.raises(ValueError, match='Did not find any files'):
+        sunpy.timeseries.TimeSeries(invalid_filepath, allow_errors=True)
 
-
+@pytest.mark.filterwarnings('ignore:"silence_errors" was deprecated in version 5')
 def test_invalid_file():
     invalid_filepath = os.path.join(rootdir, 'annotation_ppt.db')
     with pytest.raises(NoMatchError):
         sunpy.timeseries.TimeSeries(invalid_filepath)
     # Now with silence_errors kwarg set
-    ts = sunpy.timeseries.TimeSeries(invalid_filepath, silence_errors=True)
+    with pytest.warns(SunpyUserWarning):
```

Comment:
just a question - can it not just be removed from the tests? or is this something that's been decided for deprecations?

Reply:
Before time series would not actually raise any warning if a file failed to validate like this. I added the warning and this is what is being raised and caught. I will add the match message.